### PR TITLE
Munit to 1.0.0-M7

### PR DESCRIPTION
**Pull request by pacdaemon**
Bumping munit version to the latest milestone available of the 1.0.x.

### CODE REVIEW

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
-version in ThisBuild := "0.40.15-SNAPSHOT"
+ThisBuild / version := "0.40.16-SNAPSHOT"
```
Comment (dimafeng) [inline version.sbt:1]: Could you please revert this change? Versions are managed by the release plugin.

Comment (pacdaemon) [reply version.sbt]: Sure!, reverting the version.

**Comment by dimafeng** [comment]
It seems that the munit test hasn't passed
```
==> X com.dimafeng.testcontainers.munit.TestContainerForAllSpec.call afterContainersStart() and beforeContainersStop()  0.086s java.lang.NullPointerException: Cannot invoke "scala.collection.IterableOnce.knownSize()" because "coll" is null
    at scala.collection.mutable.ArrayBuffer$.from(ArrayBuffer.scala:271)
    at scala.collection.mutable.ArrayBuffer$.from(ArrayBuffer.scala:266)
    at scala.collection.IterableFactory.apply(Factory.scala:103)
    at scala.collection.IterableFactory.apply$(Factory.scala:103)
    at scala.collection.mutable.ArrayBuffer$.apply(ArrayBuffer.scala:266)
    at munit.MUnitRunner.munitTests(MUnitRunner.scala:48)
    at munit.MUnitRunner.runAll(MUnitRunner.scala:174)
    at munit.MUnitRunner.runAsync(MUnitRunner.scala:121)
    at munit.MUnitRunner.run(MUnitRunner.scala:116)
    at com.dimafeng.testcontainers.munit.TestContainerForAllSpec$.com$dimafeng$testcontainers$munit$TestContainerForAllSpec$$$run(TestContainerForAllSpec.scala:177)
    at com.dimafeng.testcontainers.munit.TestContainerForAllSpec.$init$$$anonfun$4(TestContainerForAllSpec.scala:56)
```

**Comment by pacdaemon** [comment]
I missed the failing test locally; I'll dig into it.

**Comment by rssh** [comment]
submitted workaround in #247

**Comment by pacdaemon** [comment]
> I missed the failing test locally; I'll dig into it.



> submitted workaround in #247

Nice, thanks. That works. Maybe I can close my PR. Do we need to update the release notes as here [release-notes.md](https://github.com/testcontainers/testcontainers-scala/pull/245/files#diff-82200ce47fa528129da57d8979b63119ebdc262e3d07cb7882fdb8d1ecdf4a00R4)?

**Comment by dimafeng** [comment]
closing this as it was solved in https://github.com/testcontainers/testcontainers-scala/pull/247
